test/js/components/tasks/cards.test.jsx

```diff
@@ -443,6 +442,166 @@ describe('<OrgCards/>', () => {
+      });
+
+      test('currently submitting', () => {
+        const { getByText } = setup({
+          task: {
+            ...defaultTask,
+            commits: [],
+            pr_is_open: true,
+            currently_submitting_review: true,
+          },
+          orgs: {
+            ...orgs,
+            QA: { ...orgs.QA, latest_commit: 'parent', has_been_visited: true },
+          },
+        });
+
+        expect(getByText('Submitting Review…')).toBeVisible();
+      });
+
+      describe('submit review btn', () => {
+        test('enabled', () => {
+          const { getByText } = setup({
+            task: {
+              ...defaultTask,
+              commits: [],
+              pr_is_open: true,
+              review_valid: true,
+            },
+            orgs: {
+              ...orgs,
+              QA: {
+                ...orgs.QA,
+                latest_commit: 'parent',
+                has_been_visited: true,
+              },
+            },
+          });
+
+          expect(getByText('Update Review')).toBeVisible();
+        });
+
+        test('disabled', () => {
+          const { getByText } = setup({
+            task: {
+              ...defaultTask,
+              commits: [],
+              pr_is_open: true,
+              review_valid: false,
+            },
+            orgs: {
+              ...orgs,
+              QA: {
+                ...orgs.QA,
+                latest_commit: 'parent',
+                has_been_visited: false,
+              },
+            },
+          });
+          fireEvent.focus(getByText('Submit Review'));
+
+          expect(getByText('Submit Review')).toHaveAttribute('disabled');
+        });
+      });
+
+      describe('orgStatus', () => {
+        test('renders "Approved" status', () => {
+          const { getByText } = setup({
+            orgs: {
+              ...orgs,
+              QA: {
+                ...orgs.QA,
+                latest_commit: 'parent',
+                has_been_visited: true,
+                url: 'url',
+              },
+            },
+            task: {
+              ...defaultTask,
+              commits: [],
+              review_status: 'Approved',
+              review_valid: true,
+            },
+          });
+          // debug();
+          expect(getByText('Approved')).toBeVisible();
+        });
+
+        test('renders "Changes requested" status', () => {
+          const { getByText } = setup({
+            orgs: {
+              ...orgs,
+              QA: {
+                ...orgs.QA,
+                latest_commit: 'parent',
+                has_been_visited: true,
+                url: 'url',
+              },
+            },
+            task: {
+              ...defaultTask,
+              commits: [],
+              review_status: 'Changes requested',
+              review_valid: true,
+            },
+          });
+
+          expect(getByText('Changes requested')).toBeVisible();
+        });
```

Comment:
@jgerigmeyer same with this test, unsure of the util for testing multi-lined text:

```
      Changes requested
                         | Submitted review 4 months ago
```